elytron/WFCORE-4496-principal-transformer-in-aggregate-realm.adoc

```diff
@@ -0,0 +1,71 @@
+= [WFCORE-4496] Adding principal-transformer in aggregate-realm
+:author:            Ashley Abdel-Sayed
+:email:             aabdelsa@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+Elytron currently has an aggregate security realm which is a combination of two or more realms: an authentication realm
+and one or more authorization realms. There is currently no way to transform the principal in between loading the authentication
+identity and loading the authorization identity.
+
+This enhancement would allow transformations to be applied to the principal after loading the authentication identity
```

Comment:
Who, where and how will be defined the principal-transformer, please?
Can you write some sentences about it?

Reply:
Added

Reply:
+1